python/peacock/ExodusViewer/plugins/GoldDiffPlugin.py

```diff
@@ -89,25 +80,30 @@ def __init__(self, size=None):
         self.MainLayout.addWidget(self.LinkToggle)
 
         self.GoldVTKWindow = ExternalVTKWindowPlugin(self.GoldToggle, size=size, text='GOLD')
-        self.DiffVTKWindow = ExternalVTKWindowPlugin(self.DiffToggle, size=size, text='EXODIFF')
-
-        self.setup()
-        self.setCollapsed(True)
 
         # Locate MOOSE exodiff program
         self._exodiff = None
         moose_dir = os.path.abspath(os.path.join(os.path.realpath(inspect.getfile(self.__class__)), '..', '..', '..'))
```

Comment:
This is what was causing the crash on linux. It is actually reproducible on mac as well. You just need
to `unset MOOSE_DIR` then 
`python/peacock/peacock -e modules/combined/combined-devel -i python/peacock/tests/common/simple_diffusion.i --run`

Adding  `"..", ".."` to the statement seems to fix the immediate issue but the case where exodiff is not found will still break.

Reply:
Thanks for finding this, I will figure it out next week or you can just push on top of this if you want to take care of it.